data/remnant missions.txt

```diff
@@ -495,3 +560,718 @@ mission "Remnant: Technology Available"
+				accept
+	
+	on complete
+		event "remnant: sign studies complete" 20 40
+		log "Have been introduced to the basics of the Remnant sign language, and formally granted access to their worlds after having been genetically tested on each of them."
+		conversation
+			`The last leg of the journey was spent watching some ancient show about a merchant captain making his way through the galaxy with an unlikely crew, re-enacted by a cast of Remnant who sing and sign their way through the dialogs with exagerated simplicity. It is oddly familiar, but you can't quite place it. You notice that it seems to have a lot of subtext about the dangers of manipulation and government secrecy, especially surrounding genetics.`
+			`	You arrive at the Director's desk and offer to return the datapad to the Director, carefully signing your thanks. She seems pleased at your promising start, and encourages you to continue practicing.`
+			`	She pauses for a moment, then switches to singing. "You are probably wondering about all the blood tests. Long ago it was decided that being infiltrated by Alpha agents was a big risk, and we couldn't depend on our interstellar network to remain secure. So we implemented a policy where the first time a Remnant is granted formal access to a new planet their purity and identity is verified. Thus even if someone hacked our networks to give themselves a clean identity, they would still be caught when they started working on a new planet. We also don't broadcast that requirement on our networks, simply enforce it locally. It isn't foolproof, but it adds an additional layer of safety."`
+			`	After you finish your conversation with her, you stroll off to look around the spaceport, enjoying the fact that you can now understand a bit of the conversations going on around you. It is only the most basic elements, but it is a good start. You realize, however, that you have a lot of practice to do before you become fluent.`
+
+event "remnant: sign studies complete"
+
+
+
+mission "Remnant: Learn Sign 3"
+	name "Conversational in Remnant Sign"
+	description `Return to <destination> to see how well you can sign.`
+	destination "Caelian"
+	invisible
+	to offer
+		has "event: remnant: sign studies complete"
+	on complete
+		log "Have achieved a degree of fluency in the Remnant sign language sufficient for general conversation."
+		dialog
+			`As you make your way into the Caelian spaceport, it occurs to you that the information director who gave you the first instructional videos might appreciate how much you have learned over the past few weeks of practice. You stretch your fingers and stop by the desk, where you are able to have a simple conversation with the information director.`
```

Comment:
This could be made more generic such that the player can receive this mission for any world. Given that this mission is suppose to trigger as a response to an event, I'd make it occur on landing, then have this convo be something along the lines of `You've learned quite a bit of Remnant sign language by studying over the month.`, and then you find an info desk and have a short convo with the director to show what you've learned.